<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INMFVFXK3Q8HV22" ignored="0" tests="7" failures="3" timestamp="2022-09-17T06:57:52 IST" time="37.897" errors="0">
  <testcase name="checkPostApiWorks" time="7.398" classname="testCases"/>
  <testcase name="checkGetstatsApiWorks" time="0.12" classname="testCases"/>
  <testcase name="checkAverageTimeIsCorrect" time="5.074" classname="testCases">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [5030] but found [722316]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [5030] but found [722316]
at testCases.checkAverageTimeIsCorrect(testCases.java:60)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- checkAverageTimeIsCorrect -->
  <testcase name="checkGetsApiWorks" time="5.067" classname="testCases"/>
  <testcase name="checkpostAPIResponseTime" time="5.032" classname="testCases">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [2000] but found [5023]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [2000] but found [5023]
at testCases.checkpostAPIResponseTime(testCases.java:87)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- checkpostAPIResponseTime -->
  <testcase name="checkencodedpassword" time="5.05" classname="testCases">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [ZjM5NGY1OTE4NjEyY2Q1MjM4YzQ2YmZkZTI1NjVhZTlhNjdiMzhjNg==] but found [T3PFZh3ZFx5rtry3taC8m8GYZIshx2Cx1wwYYhl5zXyazFq0RAa0uM5a24SBoLGZWQ9CYSLo4uVXrZFci/jrnw==]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [ZjM5NGY1OTE4NjEyY2Q1MjM4YzQ2YmZkZTI1NjVhZTlhNjdiMzhjNg==] but found [T3PFZh3ZFx5rtry3taC8m8GYZIshx2Cx1wwYYhl5zXyazFq0RAa0uM5a24SBoLGZWQ9CYSLo4uVXrZFci/jrnw==]
at testCases.checkencodedpassword(testCases.java:99)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- checkencodedpassword -->
  <testcase name="check2SimultaneousPostApi" time="10.077" classname="testCases"/>
</testsuite>
